runtime/parser/parse_connector.go

```diff
@@ -65,19 +71,23 @@ func (p *Parser) parseConnector(node *Node) error {
 	// NOTE: After calling insertResource, an error must not be returned. Any validation should be done before calling it.
 
 	r.ConnectorSpec.Driver = tmp.Driver
-	r.ConnectorSpec.Properties = tmp.Defaults
+	r.ConnectorSpec.Properties = propertiesPB
 	r.ConnectorSpec.TemplatedProperties = templatedProps
 	r.ConnectorSpec.Provision = provision
 	r.ConnectorSpec.ProvisionArgs = provisionArgsPB
 
 	return nil
 }
 
-// analyzeTemplatedProperties returns a slice of map keys that have a value which contains templating tags.
```

Comment:
Why remove this comment? Ideally every non-trivial function, especially in large packages, should have a docstring